products/ha_memgraph/v0.15.0/how_to_guides/query-ha-memgraph-programmatically.md

```diff
@@ -0,0 +1,222 @@
+## How to Query HA Memgraph Programmatically?
+
+If you have successfully
+[configured HA Memgraph](configure-and-run-ha-memgraph.md) along with the proxy
+setup, the process of programmatic querying should be exactly the same as
+it would be to query the single node Memgraph instance.
+
+### Supported Languages
+
+If users wish to query HA Memgraph programmatically, they can do so using the
+[Bolt protocol](https://boltprotocol.org). Bolt was designed for efficient
+communication with graph databases and Memgraph supports
+[Version 1](https://boltprotocol.org/v1) of the protocol. Bolt protocol drivers
+for some popular programming languages are listed below:
+
+  * [Java](https://github.com/neo4j/neo4j-java-driver)
+  * [Python](https://github.com/neo4j/neo4j-python-driver)
+  * [JavaScript](https://github.com/neo4j/neo4j-javascript-driver)
+  * [C#](https://github.com/neo4j/neo4j-dotnet-driver)
+  * [Ruby](https://github.com/neo4jrb/neo4j)
+  * [Haskell](https://github.com/zmactep/hasbolt)
+  * [PHP](https://github.com/graphaware/neo4j-bolt-php)
+
+### Secure Sockets Layer (SSL)
+
+Secure connections are supported and enabled by default. The server initially
+ships with a self-signed testing certificate. The certificate can be replaced
+by editing the following parameters in `/etc/memgraph/memgraph_ha.conf`:
```

Comment:
The configuration parameter is actually `/etc/memgraph/memgraph_ha_proxy.conf` because the clients communicate to the proxy directly and not to the cluster.